### PR TITLE
Added PR and issue templates according to plots2

### LINKED ISSUE #159: Add PR and issue template in the .github folder
# Description
Currently the .github folder of the repo has only a FTO issue template.
# Solution
The regular issue template and PR template can be added here like other public lab repositories.
https://github.com/publiclab/plots2/blob/master/.github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md
https://github.com/publiclab/plots2/blob/master/.github/PULL_REQUEST_TEMPLATE.md

If approved I'd like to add these templates in their respective directories
Thanks!

### PR DESCRIPTION
**Pull request by ananyaarun**
Fixes #159 

-Added PR and issue templates following the directory structure of plots2 repo.
-Updated the msg of the bot for newcomers according to plots2.

### CODE REVIEW

**Comment by ananyaarun** [comment]
@jywarren , @gauravano I added PR and issue templates following the plots2 structure.
Also i noticed that the bot msg for newcomers had been updated in a recent PR in the config.yml and first-timers.yml files.
I changes those accordingly too.

Since we made this change in plots2 I think we can create FTO's like this publiclab/plots2#3681, maintain a checklist and update all the public lab repos to follow this structure of templates and bot messages. 
Thoughts on this ?

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+
+> What did you expect to see that you didn't?
+
+
+
+### Please show us where to look
+
+https://publiclab.org/ ...
```
Comment (gauravano) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]:
```suggestion
https://publiclab.org/ ... or https://publiclab.github.io/leaflet-environmental-layers/example
```

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+### What's your PublicLab.org username?
```
Comment (gauravano) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]: I think we can remove this one?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Fixes #0000 (<=== Add issue number here)
+
+Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!
+
+* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
+* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
```
Comment (gauravano) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
```

File: .github/config.yml [hunk 4]
```diff
@@ -4,22 +4,37 @@
...
+  Do join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions. 
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
- Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
- There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+ Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
+ `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
+ Also please refer [here](https://github.com/publiclab/plots2/issues/3840) for installation help 💿
```
Comment (gauravano) [inline .github/config.yml]:
```suggestion
```

File: .github/config.yml [hunk 5]
```diff
@@ -4,22 +4,37 @@
...
+ 
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: |
- Congrats on merging your first pull request! 🙌🎉⚡️ 
- Your code will likely be published to https://publiclab.github.io/leaflet-environmental-layers/example/ in the next few days. 
- In the meantime, can you tell us your Twitter handle so we can thank you properly?
+ Congrats on merging your first pull request! 🙌🎉⚡️
+ Your code will likely be published to PublicLab.org in the next few days, but first it will be published to https://stable.publiclab.org/ (it will take some minutes for this to load, and until then you may see logs from the build process). Please test out your work on this testing server and report back with a comment that all has gone well!
```
Comment (gauravano) [inline .github/config.yml]:
```suggestion
 Your code will likely be published to https://publiclab.github.io/leaflet-environmental-layers/example/ in the next few days.
```

File: .github/first-timers-issue-template.md [hunk 6]
```diff
@@ -4,7 +4,7 @@ If that's you, we're interested in helping you take the first step and can answe
 
 We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
 
-If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/leaflet-environmental-layers/labels/help-wanted) issues. Thanks!
+If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!
```
Comment (gauravano) [inline .github/first-timers-issue-template.md]:
```suggestion
If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/leaflet-environmental-layers/labels/help-wanted) issues. Thanks!
```

**Comment by ananyaarun** [comment]
@gauravano , should have noticed the plots2 instead of LEL in all those templates !!
I made the required changes now :)
Do u think we should do this for all the repos too ?

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (ananyaarun) [reply .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE.md]: yes it seems redundant here !

**Comment by ananyaarun** [comment]
@gauravano , I made the required changes. should have noticed the plots2 instead of LEL in my previous commit !!
Also do u think its a good idea to add these templates to other repos as well ?

**Comment by gauravano** [comment]
And, yes we can do these changes to other repos too as it will help contributors. Thanks!

**Comment by ananyaarun** [comment]
@gauravano , we can make a checklist on say plots2 to cover all remaining repos and break them into several FTO's ? Shall i do that ?

**Comment by gauravano** [comment]
Sure go ahead. Also, at some repos, there might be issues already open in this regard so do check that before including that repo name in your checklist. 

Thanks @ananyaarun!

**Comment by jywarren** [comment]
Thank you!!!!! 🎉 🙌

**Comment by ananyaarun** [comment]
@gauravano @jywarren is this still relevant ?
If so shall we update all the templates according to the work done by @sashadev-sky ?

**Comment by sashadev-sky** [comment]
@ananyaarun Are you referring to the issue templates in LDI I made? I spoke to Jeff last week about those, he said he liked them and would be ok with implementing them in other projects, but it's not a priority. If you are interested in updating for LEL go for it!

**Comment by ananyaarun** [comment]
Yes @sashadev-sky , The ones in LDI
Sure if it is nor priority we can put this on hold.

**Pull request merged**